content/kotlin/concepts/arrays/map/map.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.map()'
+Description: 'map() is a Kotlin method that transforms elements in a collection.'
+Subjects: 
+  - 'Code Foundations'
+  - 'Mobile Development'
+Tags: 
+  - 'Android'
+  - 'Arrays'
+  - 'Map'
+  - 'Kotlin'
+CatalogContent: 
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+**map()** is a Kotlin method that transforms elements in a collection (Array, List, or Set) using the provided logic. It does not modify the original collection, which can be useful when data permanence is required.
```

Comment:
```suggestion
**`map()`** is a Kotlin method that transforms elements in a collection (Array, List, or Set) using the provided logic. It does not modify the original collection, which can be useful when data permanence is required.
```
Since .map( ) is a function therefore it should be included in single backticks (`.....`).

Reply:
I've applied this change locally.